include/envoy/server/listener_manager.h

```diff
@@ -13,13 +13,34 @@
 namespace Envoy {
 namespace Server {
 
+/**
+ * Interface for an LDS API provider.
+ */
+class LdsApi {
+public:
+  virtual ~LdsApi() {}
+
+  /**
+   * @return the last received version by the xDS API for LDS.
```

Comment:
Nit on return type in @return. I think we're at the point that this is not being enforced consistently..